From 9fa8141cf5e19f469c27c218c7378e9b3e87cf2e Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Mon, 4 May 2020 17:27:01 -0700
Subject: [PATCH] dgit-maint-{merge,rebase}(7): See also git-revisions(7)

For help with some of the advanced git commands presented.

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 dgit-maint-debrebase.7.pod | 2 +-
 dgit-maint-merge.7.pod     | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/dgit-maint-debrebase.7.pod b/dgit-maint-debrebase.7.pod
index 373fb2f7..67ef0870 100644
--- a/dgit-maint-debrebase.7.pod
+++ b/dgit-maint-debrebase.7.pod
@@ -781,7 +781,7 @@ See also "ILLEGAL OPERATIONS" in git-debrebase(5).
 
 =head1 SEE ALSO
 
-dgit(1), dgit(7), git-debrebase(1), git-debrebase(5)
+dgit(1), dgit(7), git-debrebase(1), git-debrebase(5), git-revisions(7)
 
 =head1 AUTHOR
 
diff --git a/dgit-maint-merge.7.pod b/dgit-maint-merge.7.pod
index c3931ebe..d675a6dd 100644
--- a/dgit-maint-merge.7.pod
+++ b/dgit-maint-merge.7.pod
@@ -526,7 +526,7 @@ next push will then require I<--overwrite>.
 
 =head1 SEE ALSO
 
-dgit(1), dgit(7)
+dgit(1), dgit(7), git-revisions(7)
 
 =head1 AUTHOR
 
-- 
2.26.2

